ext/mysqlnd/mysqlnd_vio.c

```diff
@@ -414,6 +414,29 @@ MYSQLND_METHOD(mysqlnd_vio, set_client_option)(MYSQLND_VIO * const net, enum_mys
 			net->data->options.ssl_verify_peer = val;
 			break;
 		}
+		case MYSQL_OPT_TLS_VERSION:
+			if (strcmp(value, "TLSv1") == 0) {
```

Comment:
looks like code duplication https://github.com/php/php-src/pull/2675/files#diff-07a72755040c3a5d02660c0559d48a3bR820
I would suggest to move it to separate function/macro